app/assets/stylesheets/views/_landing_page/side-navigation.scss

```diff
@@ -0,0 +1,7 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.side-nav {
+  @include govuk-media-query($until: desktop) {
+    display: none;
+  }
+}
```

Comment:
Nitpick: GitHub doesn't seem to like it when there's no empty line at the end of files, if you'd like to fix that.